tests/test_spark.py

```diff
@@ -175,6 +175,18 @@ def test_jars_flag(service_name=utils.SPARK_SERVICE_NAME):
         args=["--jars {}".format(uploadedJarUrl),
               "--class MultiConfs"])
 
+@pytest.mark.sanity
+def test_packages_flag(service_name=utils.SPARK_SERVICE_NAME):
+    utils.run_tests(
+        app_url=utils.dcos_test_jar_url(),
+        app_args="20",
+        expected_output="The sum of 20 numbers is 210",
+        service_name=service_name,
+        args=["--conf spark.mesos.containerizer=mesos",
+              "--conf spark.jars.ivy=/mnt/mesos/sandbox/.ivy2",
```

Comment:
This way users have to provide `--conf spark.jars.ivy` every time they use `--packages` with the CLI. The target goal is to specify it add this property in `dcos spark cli` when `--packages` is specified. Let's use `$MESOS_SANDBOX` env var instead of hardcoding as well.

Reply:
No need to specify `spark.jars.ivy` every time when using `--packages` flag. It will be added automatically.